src/modules/registrypreview/RegistryPreviewUI/MainWindow.xaml

```diff
@@ -41,166 +52,135 @@
             Height="Auto"
             x:FieldModifier="public"
             Loaded="GridPreview_Loaded"
-            TabFocusNavigation="Cycle">
+            TabFocusNavigation="Cycle"
+            Margin="16,0,16,16">
             <Grid.Resources>
-                <Style x:Key="GridCardStyle" TargetType="Grid">
+                <Style x:Key="GridCardStyle" TargetType="Border">
                     <Style.Setters>
                         <Setter Property="Background" Value="{ThemeResource CardBackgroundFillColorDefaultBrush}" />
                         <Setter Property="BorderThickness" Value="1" />
                         <Setter Property="BorderBrush" Value="{ThemeResource CardStrokeColorDefaultBrush}" />
+                        <Setter Property="CornerRadius" Value="{StaticResource OverlayCornerRadius}"/>
                     </Style.Setters>
                 </Style>
             </Grid.Resources>
             <Grid.ColumnDefinitions>
                 <!--  Left, Splitter, Right  -->
                 <ColumnDefinition Width="*" />
-                <ColumnDefinition Width="6" />
+                <ColumnDefinition Width="8" />
                 <ColumnDefinition Width="*" />
             </Grid.ColumnDefinitions>
             <Grid.RowDefinitions>
-                <!--  CommandBar, Tree, Splitter, List, StackPanel  -->
+                <!--  CommandBar, Tree, Splitter, List  -->
                 <RowDefinition Height="Auto" />
                 <RowDefinition Height="*" />
-                <RowDefinition Height="6" />
+                <RowDefinition Height="8" />
                 <RowDefinition Height="*" />
             </Grid.RowDefinitions>
 
-            <Grid
+            <CommandBar
```

Comment:
Please put back the background color of the `CommandBar`.. this was per design specification.

Reply:
We have design syncs with the design team where we discuss these things. They showed this as an example (Outlook):

<img width="875" alt="image" src="https://github.com/microsoft/PowerToys/assets/9866362/78edde64-6a96-4cc2-9465-0eb26d3e1e66">

Reply:
Oh I see... The Outlook family - and Office family - do have a different UI then most (MS Store) apps. But if you want me to add it back I will. You're saying this style is going to be applied to "all" other MS apps in the future?

Reply:
Please do. W11 introduced the notion of "cards" - and something that is part of the broader Fluent Design story.

For best practices, and the latest guidance, the Windows Visual Library in Figma is your best source: https://www.figma.com/community/file/1159947337437047524/Windows-UI-3

As a rule of thumb, the content there is the latest defined by the design team owning the W11 design language. Some of the content you will not find in, or might contradict, the documentation on MS Docs. Those articles sometimes refer to the UWP/W10 days, and are mostly targeting < WinUI 2.5 (before the design refresh)

Reply:
Detail: a commandbar is by default aligned to the right. Setting `HorizontalAlignment="Left"` will break the "stretch" property, and then an extra border is visible when the overflow menu is open. Still want to keep it left? Right?
![image](https://github.com/microsoft/PowerToys/assets/65828559/418cdb22-9916-40a0-8ed2-572902d04d92)